packages/utils/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.9.0] - 2024-03-28
+### Fixed
+- Fix dictionary tests importing from one another, fix not handling v2 block range type (#2322)
```

Comment:
I don't think changelog messages should refer to tests only changes that are user facing